rules-tests/Symfony73/Rector/Class_/InvokableCommandRector/Fixture/some_command.php.inc

```diff
@@ -14,6 +14,7 @@ final class SomeCommand extends Command
 {
     public function configure()
     {
+        $this->setHelp('argument');
```

Comment:
Could you add a new test fixture file only with `setHelp()` issue? 
That way we keep test fixtures separated by case. This makes them easier to maintain/debug later on

Reply:
I added `some_command_with_set_help.php.inc` where its the same as here but then explicitly name this test as we need to have `$this->addArgumen()` and   `$this->addOption()` for the `InvokableCommandRector` to remove the `configure` bug like before